### PR TITLE
Update and rename dependent/american-journal-of-hypertension.csl to a…

### PR DESCRIPTION
**Pull request by damnation333**
via https://github.com/citation-style-language/styles/issues/2705

### CODE REVIEW

File: american-journal-of-hypertension.csl [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                    <date-part name="month" suffix=" "/>
+                    <date-part name="year"/>
+                  </date>
+                </else>
+              </choose>
+              <text variable="volume" prefix=" "/>
+            </group>
+            <text variable="page" prefix=":"/>
+          </group>
+          <text variable="DOI" prefix=" &lt;" suffix="&gt;"/>
```
Comment (rmzelle) [inline american-journal-of-hypertension.csl]: @damnation333, where did you find this? I e.g. found this open access article, which doesn't print DOIs at all (https://academic.oup.com/ajh/article/30/7/683/3739803/An-Assessment-of-the-Accuracy-of-Home-Blood), and the instructions also don't seem to mention them (https://academic.oup.com/ajh/pages/general_instructions#FormatofManuscripts).

Unless they're very clear about this, I'd just use the URL format here.

**Comment by damnation333** [comment]
@rmzelle 
I had seen it here: Reference 4 https://academic.oup.com/ajh/article/30/6/587/3064943/Association-of-Blood-Pressure-Trajectory-With

**Comment by rmzelle** [comment]
@damnation333, they still add the URL prefix there: `<http://dx.doi.org/10.1001/jama.2016.7050>`, in addition to the angle brackets. So I would go with `<https://doi.org/...>` here (especially since they already use ` DOI: https://doi.org/10.1093/ajh/hpx028` elsewhere on the page.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**